domain/src/test/kotlin/no/nav/su/se/bakover/domain/behandling/BehandlingsinformasjonTest.kt

```diff
@@ -172,31 +169,17 @@ internal class BehandlingsinformasjonTest {
         val underMinstegrense = BeregningStrategy.BorAlene.beregn(
             Beregningsgrunnlag(
                 periode = periode,
-                fradrag = listOf(
-                    FradragFactory.ny(
-                        periode = periode,
-                        type = Fradragstype.ForventetInntekt,
-                        beløp = Sats.HØY.månedsbeløp(1.januar(2020)) - 200,
-                        utenlandskInntekt = null,
-                        tilhører = FradragTilhører.BRUKER
-                    )
-                )
+                forventetInntektPerÅr = Sats.HØY.årsbeløp(1.januar(2020)) - 200,
```

Comment:
Jeg er litt mer fan av testdata som ikke er relatert til implementasjonen :shrug: I tillegg kunne man kanskje brukt grenseverdiene. Beste hadde kanskje vært en form for property testing, men jeg tror ikke junit5 støtter det default

Reply:
Litt lat her tenker jeg, ved andre øyekast tenker jeg også at denne testen ligger litt malplassert? Tenker at jeg kan sikte på en mer presis test ved å refaktorere avslagslogikken i beregningen og teste den isolert.

Reply:
Fikser i ny pr.